app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -1,29 +1,14 @@
 # frozen_string_literal: true
 
 class OmniauthCallbacksController < Devise::OmniauthCallbacksController
-  def google_oauth2
-    if user.present?
-      user.accept_invitation! if invitation_token
-      upload_avatar(omniauth_avatar)
-
-      flash[:notice] = I18n.t('devise.omniauth_callbacks.success',
-                              kind: t('omniauth.google'))
-      sign_in_and_redirect @user, event: :authentication
-    else
-      redirect_to new_user_session_path, notice: t('omniauth.access_denied')
-    end
-  end
-
-  def facebook
-    if user.present?
-      user.accept_invitation! if invitation_token
-      upload_avatar(omniauth_avatar)
-      flash[:notice] = I18n.t('devise.omniauth_callbacks.success',
-                              kind: 'Facebook')
-      sign_in_and_redirect @user, event: :authentication
-    else
-      redirect_to new_user_session_path, notice: t('omniauth.access_denied')
-    end
+  def omniauth_login(provider)
```

Comment:
You'll need to update the tests since `google_oauth2` no longer exists. That's why all those tests are failing in the CircleCI build. 

I really recommend running tests locally and not depending on CircleCI.